src/_includes/docs/install/compiler/xcode.md

```diff
@@ -98,7 +92,26 @@ To [install and set up CocoaPods][cocoapods], run the following commands:
    {{prompt1}} brew install cocoapods
    ```
 
-Using Homebrew reduces potential issues with chipsets and install permissions.
```

Comment:
@jmagman Why do we recommend Homebrew, but the [CocoaPods Getting Started guide](https://guides.cocoapods.org/using/getting-started.html) says to use `gem install` with the system Ruby installation? 

@atsansone It would be great if we can lean more on the CocoaPods documentation here. At the very least, we should try to match what they recommend.

Reply:
The reason I included Homebrew is that it was included in a previous version of this doc for the reasons stated in the Software requirements: Homebrew handles chipset and permissions issues for you. I would love to avoid: 

1. Sending users to too many other sites.
2. Needing to cover off more edge cases when something doesn't work.

Reply:
@johnpryan I wouldn't have suggested Homebrew, I wasn't a reviewer on https://github.com/flutter/website/pull/9736.  

https://docs.flutter.dev/get-started/install/macos/mobile-ios#install-cocoapods currently sounds like Homebrew is a requirement to do Flutter iOS development it just isn't.  cc @christopherfujino 

I don't use Homebrew to manage Ruby or CocoaPods.  I use the built in Ruby version to macOS and I follow CocoaPod's own installation instructions.  For the purpose of this doc I would personally just point them to https://guides.cocoapods.org/using/getting-started.html#installation and call it a day instead of trying to replicate dependency's installation instructions.

Reply:
I'm also not a fan of Homebrew and skip any steps involving it. Leaning on Cocoapods own installation instructions and the options there seems good 👍

Reply:
I got lost searching through the git history, but TL;DR we used to recommend brew install cocoapods, but this caused pain for our users, and is NOT the official way to install this, so we standardized on `gem install cocoapods` across the tool and website as our official recommendation.

Reply:
> For the purpose of this doc I would personally just point them to https://guides.cocoapods.org/using/getting-started.html#installation and call it a day instead of trying to replicate dependency's installation instructions.

This is aligned with what the tool tells users to do if we detect it's either missing or broken: https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/macos/cocoapods.dart#L50

Reply:
Most any explanation I've consulted doesn't recommend the default macOS install of Ruby. We used Homebrew elsewhere, so it seemed better to have one way to install everything instead of one way for this app, another for this other app, etc. Think as a new user: keep it simple.

We refer to Homebrew in about as many pages in the public docs: around 1-2 times each.

Looking at this as a new user, I'd like to recommend simple, consistent ways to handle installs wherever possible. Homebrew offers that option.

Further, the CocoaPods install docs themselves are out of date (referring to `bash_profile`, for example). 

Although, @christopherfujino , your point about what exists in the code makes sense and may need a future revisit.

Reply:
@jmagman : Understood that CocoaPods is not required to build iOS apps for Flutter. It's required to build Flutter plugins for iOS apps, right? We could make that note, but not in a Getting Started. I would then need to add an additional section or page later to cover that off. What say you?

Reply:
CocoaPods is required as soon as you add a plugin.  I believe in the docs that used to be called out.

Reply:
It still is. ;)